lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -23,9 +23,9 @@
 using System.Threading;
```

Comment:
The `public` property `Client` is only used within the class and only assigned to in the constructor. It is probably safe to replace it with a `private readonly` field.